src/views/product-tutorials-view/helpers/heading-helpers.ts

```diff
@@ -53,13 +50,39 @@ function buildLayoutHeadings(
   return [...overviewHeading, ...blockHeadings, ...sitemapHeading]
 }
 
+/**
+ * Check if a block has a heading,
+ * and is a block type we want to show in the table of contents,
+ * in which case we can generate a headingSlug for it
+ */
+function isIntendedForTableOfContents<
```

Comment:
I've update to be more explicit about which blocks are intended for inclusion in our table of contents.